### DISCUSSION TITLE
Certyfikat SSL odpowiedni dla Azure Front Door

### DISCUSSION THREAD

**Discussion by miqm** [AzureCommunityPL/group#95] [Pomoc] [answered]
Cześć,

Czy certyfikat rapidSSL/thawte/geotrust kupiony na ssl24.pl będzie działać z Azure Front Door? Dema certyfikatów wskazują jako Issuer RapidSSL TLS RSA CA G1 -> DigiCert Global Root G2. Obu nie ma na liście dozwolonych wystawców https://docs.microsoft.com/en-us/azure/frontdoor/front-door-troubleshoot-allowed-ca

Pytanie czy faktycznie one nie zostaną przyjęte, czy lista nie jest aktualna a DigiCert Global Root G2 jest jednak uznawany pomimo jego braku na liście (a nawiasem mówiąc, certyfikat *.azureedge.net jest autoryzowany przez DigiCert Global Root G2)

Odpowiadając na pytania - nie mogę użyć FD Managed gdyż potrzebuję zabezpieczyć główną domenę, na którą nie mogę dać CNAME.

**Comment by miqm** [comment, marked as answer]
Mały follow-up. Ostatecznie znalazłem dostawcę dającego cert podpisany przez Digicert Global Root CA - RapidSSL od gogetssl.com

Równolegle otworzyłem ticket aby MS zweryfikował, czy to CA (G2) jest akceptowane i poprawił dokumentację. https://github.com/MicrosoftDocs/azure-docs/issues/63779

**Reply by wi5nia** [reply]
Dziś też z supportem wymieniałem ticketa żeby tę listę zaktualizowali 😄
